src/net.rs

```diff
@@ -636,6 +770,26 @@ impl Actor {
         }
         Ok(())
     }
+
+    async fn handle_receiver_gone(
+        &mut self,
+        topic: TopicId,
+        receiver_id: ReceiverId,
+    ) -> anyhow::Result<()> {
+        if let Entry::Occupied(mut entry) = self.topics.entry(topic) {
+            let state = entry.get_mut();
+            state.event_senders.remove(&receiver_id);
+            if !state.still_needed() {
+                self.quit_queue.push_back(topic);
+                self.process_quit_queue().await?;
+            }
+        } else {
+            // topic should not have been dropped without all receivers being dropped first
+            warn!("receiver gone for missing topic");
```

Comment:
maybe log the topic, so there is a chance to figure out what's going on

Reply:
done in 6138e47